src/styles/utils/mixins.scss

```diff
@@ -0,0 +1,5 @@
+@mixin card-button-hovered {
+  border: 1px solid $card-button-hovered-color;
```

Comment:
can we move this to a `button` class?
we can use `box-sizing: border-box` for it
change on hover only what is needed

Reply:
@VolodymyrKirichenko  Do you mean that I need to move these styles without mixin to `card__buy-button`? Or do I need to create a separate `button` block?